app/models/relationship.rb

```diff
@@ -8,11 +8,11 @@ class Relationship < ApplicationRecord
   validates :followed_id, presence: true
   validates :status, presence: true
 
-  after_create :send_relationship_notification, unless: -> { Rails.env.test? }
+  after_create :send_relationship_notification
 
   private
 
   def send_relationship_notification
-    RelationshipMailer.new_follower(self).deliver_later
+    SendNewRelationshipEmailJob.perform_later(id)
```

Comment:
You don't need to put this into job. 
Just make sure that your config is correct :  https://gist.github.com/maxivak/690e6c353f65a86a4af9